kie-aries-blueprint/pom.xml

```diff
@@ -33,6 +33,12 @@
       <artifactId>org.apache.aries.blueprint.api</artifactId>
       <scope>compile</scope>
     </dependency>
+    <!--<dependency>-->
```

Comment:
I would vote for removing the commented out dependency, unless you have some plans with it?